Shared/UI/Views/MessageContentView/ReadIndicatorView.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  MessageReadView.swift
+//  Jibber
+//
+//  Created by Benji Dodgson on 4/19/22.
+//  Copyright © 2022 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+import ScrollCounter
+
+class ReadIndicatorView: BaseView {
+    
+    private let imageView = UIImageView(image: UIImage(systemName: "eyeglasses"))
+    private let counter = NumberScrollCounter(value: 0,
+                                              scrollDuration: Theme.animationDurationSlow,
+                                              decimalPlaces: 0,
+                                              prefix: "",
+                                              suffix: nil,
+                                              seperator: "",
+                                              seperatorSpacing: 0,
+                                              font: FontType.small.font,
+                                              textColor: ThemeColor.T1.color,
+                                              animateInitialValue: true,
+                                              gradientColor: nil,
+                                              gradientStop: nil)
+    
+    override func initializeSubviews() {
+        super.initializeSubviews()
+        
+        self.addSubview(self.imageView)
+        self.addSubview(self.counter)
+        
+        self.layer.cornerRadius = Theme.innerCornerRadius
+        self.layer.borderColor = ThemeColor.BORDER.color.cgColor
+        self.layer.borderWidth = 0.5
+        
+        self.imageView.tintColor = ThemeColor.T1withAlpha.color
+        self.imageView.contentMode = .scaleAspectFit
+    }
+    
+    func configure(with message: Messageable) {
+        UIView.animate(withDuration: Theme.animationDurationSlow) {
+             if message.canBeConsumed {
```

Comment:
Shows just glasses if you haven't read, glasses + count if its your message or if count of readers is greater than 1